### PR TITLE
SAK-47722 - Rationalize the use of jjwt and jjwt-impl

### PR DESCRIPTION
**Pull request by csev**

### CODE REVIEW

**Review by axxter99 — APPROVED** [review]

File: deploy/pom.xml [hunk 1]
```diff
@@ -1012,6 +1012,30 @@
             <artifactId>json-simple</artifactId>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>io.jsonwebtoken</groupId>
+            <artifactId>jjwt-api</artifactId>
+            <version>${sakai.jsonwebtoken.jjwt.version}</version>
```
Comment (ern) [inline deploy/pom.xml:1018]: no need for a version here

File: deploy/pom.xml [hunk 2]
```diff
@@ -1012,6 +1012,30 @@
...
+            <artifactId>jjwt-impl</artifactId>
+            <version>${sakai.jsonwebtoken.jjwt.version}</version>
+            <scope>compile</scope>
+        </dependency>
+        <dependency>
+            <groupId>io.jsonwebtoken</groupId>
+            <artifactId>jjwt-jackson</artifactId>
+            <version>${sakai.jsonwebtoken.jjwt.version}</version>
+            <scope>compile</scope>
+        </dependency>
```
Comment (ern) [inline deploy/pom.xml:1038]:
```suggestion
        <dependency>
            <groupId>io.jsonwebtoken</groupId>
            <artifactId>jjwt-api</artifactId>
            <scope>compile</scope>
        </dependency>
        <dependency>
            <groupId>io.jsonwebtoken</groupId>
            <artifactId>jjwt-impl</artifactId>
            <scope>compile</scope>
        </dependency>
        <dependency>
            <groupId>io.jsonwebtoken</groupId>
            <artifactId>jjwt-impl</artifactId>
            <scope>compile</scope>
        </dependency>
        <dependency>
            <groupId>io.jsonwebtoken</groupId>
            <artifactId>jjwt-jackson</artifactId>
            <scope>compile</scope>
        </dependency>
```

**Review by ern — APPROVED** [review]
made a small change LGTM

**Comment by csev** [comment]
Thanks - I will merge when the build finishes.

**Pull request merged**